src/data/collection.json

```diff
@@ -32,6 +32,25 @@
 		"notes": "Art shopping",
 		"badge": null
 	},
+	{
+		"url": "https://github.com/dhammon/ai-goat",
+		"name": "AI-Goat",
+		"collection": [
+			"offline"
+		],
+		"technology": [
+			"Python"
+		],
+		"references": [
+			{
+				"name": "download",
+				"url": "https://github.com/dhammon/ai-goat"
+			}
+		],
+		"author": "IBM/Watchfire",
+		"notes": "Log in with jsmith/demo1234 or admin/admin",
+		"badge": "hclproducts/AltoroJ"
```

Comment:
This doesn't seem right

Reply:
I've forked the master branch, so it was likely there already. I'm not entirely sure,.

Reply:
Yeah noticed it, sorry for the trouble.